docs/contracts/inheritance.rst

```diff
@@ -43,7 +43,7 @@ Details are given in the following example.
 
 
     contract Owned {
-        constructor() { owner = msg.sender; }
+        constructor() { owner = payable(msg.sender); }
```

Comment:
Making owner payable here really complicates the example needlessly.

Reply:
There is the `selfdestruct(owner) ` in line 59, even though it can be replaced with `selfdestruct(payable(msg.sender))` so that we don't need `owner` to be payable. Not sure which one of the two should be recommended.

Reply:
You are right, maybe it's better as it is, even though it needs more conversions.

Reply:
If there is an intent to call `selfdestruct(owner)` then it really should be defined as payable.